frontend/public/static/locale/fr/translate.ftl

```diff
@@ -206,6 +206,9 @@ notification--entity-not-found = Impossible de charger la chaîne spécifiée
 otherlocales-Translation--copy =
     .title = Copier la traduction
 
+otherlocales-Translation--header-link = 
+    .title = Open string in { $locale } ({ $code })
+
```

Comment:
This file contains French localizations, so we should either remove this block or provide an actual translation.